NGLess/StandardModules/Mappers/Bwa.hs

```diff
@@ -38,13 +40,26 @@ hasValidIndex basepath = doAllFilesExist indexRequiredFormats
                 else return False
         indexRequiredFormats = [".amb",".ann",".bwt",".pac",".sa"]
 
+-- | Checks whether we should customize bwa's indexing blocksize
+customBlockSize :: FilePath -> IO [String]
+customBlockSize path = sizeAsParam <$> getFileSize path
+
+sizeAsParam :: FileOffset -> [String]
```

Comment:
Please add a comment here with the same information as you have in the PR (from my POV, feel free to copy & paste):

BWA's default indexing parameters are quite conservative. This leads to
a small memory footprint at the cost of more CPU hours.
With large databases (~100GB) default settings require over 2 weeks of
CPU time. Increasing the default blocksize will increase the memory
footprint but will reduce indexing time 3 to 6 fold.

This patch increases the blocksize to roughly 1/10th of the filesize.
The memory footprint should be about the size of the database.

As per lh3/bwa#104 this patch may become
obsolete once this functionality is built into bwa.